sdk/core/Azure.Core/src/Pipeline/HttpPipelineTransportOptions.cs

```diff
@@ -10,9 +11,23 @@ namespace Azure.Core.Pipeline
     /// </summary>
     public class HttpPipelineTransportOptions
     {
+        /// <summary>
+        ///
+        /// </summary>
+        public HttpPipelineTransportOptions()
+        {
+            ClientCertificates = new List<CertificateCredential>();
+        }
+
         /// <summary>
         /// A delegate that validates the certificate presented by the server.
         /// </summary>
         public Func<ServerCertificateCustomValidationArgs, bool>? ServerCertificateCustomValidationCallback { get; set; }
+
+        /// <summary>
+        ///
+        /// </summary>
+        /// <value></value>
+        public IList<CertificateCredential> ClientCertificates {get;}
```

Comment:
Nit: just because the code style would otherwise do it:
```suggestion
        public IList<CertificateCredential> ClientCertificates { get; }
```